Makefile

```
@@ -173,3 +173,49 @@ server-profile-cpu:
 jaeger:
 	open http://localhost:16686
 	docker run --rm -p 5775:5775/udp -p 6831:6831/udp -p 6832:6832/udp -p 5778:5778 -p 16686:16686 -p 14268:14268 -p 9411:9411 -e COLLECTOR_ZIPKIN_HTTP_PORT=9411 jaegertracing/all-in-one:1.7
+
+e2e-setup:
+	make e2e-setup-git
+	make e2e-setup-kind
+	make e2e-setup-fluxd
+
+	@echo "\nSetup complete\n\nRun the following to continue:\n\
+	- make e2e-run-local-daemon\n\
+	- make e2e-run-local-manager\n\
+	- make e2e-do-release"
+
+e2e-teardown:
+	kind delete cluster
+
+e2e-setup-git:
+	rm -rf e2e-test/sourcegitrepo
+	mkdir -p e2e-test/sourcegitrepo/local/releases/default
+	echo "Hello World" > e2e-test/sourcegitrepo/README.md
+	cd e2e-test/sourcegitrepo;\
```

The directory name is pretty hard to head like this. Could we maybe stick with kebab casing as with `e2e-test`?

```suggestion
	rm -rf e2e-test/source-git-repo
	mkdir -p e2e-test/source-git-repo/local/releases/default
	echo "Hello World" > e2e-test/source-git-repo/README.md
	cd e2e-test/source-git-repo;\
```

This has an effect on multiple targets so maybe put in into a variable as well?

---

Yeah, that we could do. Will change that 👍